singularity/Singularity

```diff
@@ -22,7 +22,7 @@ From:centos:7.4.1708
     yum -y groupinstall "Development tools"
     yum -y install epel-release
     yum -y install net-tools telnet wget git tar which \
-        whereis bzip2 p7zip vim vi vixie-cron cronie sudo
+        bzip2 p7zip vim vi cronie sudo
```

Comment:
what's the reason for the changes in this file?

Reply:
Those packages do not exist anymore!